packages/api/package.json

```diff
@@ -31,10 +31,13 @@
         "@mikro-orm/nestjs": "^5.0.0",
         "@mikro-orm/postgresql": "^5.0.4",
         "@nestjs/common": "^9.0.0",
+        "axios": "^0.21.0",
         "reflect-metadata": "^0.1.13",
         "rxjs": "^7.0.0"
     },
     "dependencies": {
+        "@nestjs/axios": "^1.0.0",
+        "@sendinblue/client": "^3.3.1",
```

Comment:
i think this should be a peer dependency of the brevo module, shouldn't it?

Reply:
Since this is used only in the module and not in any of the comet core packages I think this needs to be a 'normal' dependency. But I am still quite unsure about the dependency sections.